### PR TITLE
feat: added twitter pixel & lotame

### PR DESCRIPTION
**Pull request by ioanna0**

### CODE REVIEW

**Comment by sndrs** [comment]
because you've used `chore` for adding twitter and lotame, no new release will be triggered. is this what you expected?

more info here https://www.conventionalcommits.org/en/v1.0.0/#summary

**Comment by sndrs** [comment]
> because you've used chore for adding twitter and lotame, no new release will be triggered. is this what you expected?

no longer true since #53 – ignore this!

File: src/third-party-tags/twitter-uwt.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// @ts-nocheck
```
Comment (sndrs) [inline src/third-party-tags/twitter-uwt.ts:1]: do you really want to disable ts for the whole file?

**Comment by mxdvl** [comment]
Could we simply ignore the `external-scripts` folder?

**Comment by ioanna0** [comment]
> Could we simply ignore the `external-scripts` folder for ESLint and TypeScript?

Great idea, i've tried that here: 
[#4d617bf](https://github.com/guardian/commercial-core/pull/41/commits/4d617bfe21de5d5ef1734d7fd8db599e993f2f28) 

it is ignoring the Eslint issues but not the TS ones 🤔

File: src/third-party-tags/twitter-uwt.ts [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import { GetThirdPartyTag } from '../types';
-import { twitterScript } from './external-scripts/twitter-script';
+import { twitterScript as insertSnippet } from './external-scripts/twitter-script';
 
 export const twitter: GetThirdPartyTag = ({ shouldRun }) => ({
 	shouldRun,
 	name: 'twitter',
-	insertSnippet: twitterScript,
+	insertSnippet: insertSnippet,
```
Comment (mxdvl) [inline src/third-party-tags/twitter-uwt.ts:7]: this is even shorter!

```suggestion
	insertSnippet,
```

**Review by mxdvl — APPROVED** [review]
👍 LGTM

**Comment by sndrs** [comment]
just realised - don't forget to add the export to the index file https://github.com/guardian/commercial-core/blob/main/src/index.ts

**Comment by ioanna0** [comment]
oh yes forgot about it ! fixed here https://github.com/guardian/commercial-core/pull/55

**Pull request merged**